typescript-selenium-webdriver/tests/index.test.ts

```diff
@@ -1,76 +1,76 @@
-        // See https://www.npmjs.com/package/selenium-webdriver for examples.
-        driver = new Builder()
-            .forBrowser('chrome')
-            .setChromeOptions(new chrome.Options().headless())
-            .build();
-    });
-
-    afterAll(async () => {
-        await driver.quit();
-    });
-
-    beforeEach(async () => {
-        // For simplicity, we're pointing our test browser directly to a static html file on disk.
-        //
-        // In a real project, you would probably use a localhost http server (Express.js, for example)
-        // and point selenium-webdriver to a http://localhost link.
-        //
-        // See https://jestjs.io/docs/en/testing-frameworks for examples.
-        const pageUnderTest = 'file://' + path.join(__dirname, '..', 'src', 'index.html');
-        await driver.get(pageUnderTest);
-
-        // Checking for a known element on the page in beforeEach serves two purposes:
-        // * It acts as a sanity check that our browser automation setup basically works
-        // * It ensures that the page is loaded before we run our accessibility scans
-        await driver.wait(until.elementLocated(By.css('h1')));
-    });
-
-    it('only contains known accessibility violations', async () => {
-        // Run an accessibility scan using axe-webdriverjs
-        const axeResults = await AxeBuilder(driver).analyze();
-
-        // Write a test expectation that accounts for "known" issues we want to baseline
-        expect(axeResults.violations.length).toBe(3);
-
-        // Write the axe results to a .sarif file, so we can use the SARIF Multitool to
-        // apply a baseline file and show the results in the Scans tab in Azure Pipelines
-        const sarifResults = convertAxeToSarif(axeResults);
-        const testResultsDirectory = path.join(__dirname, '..', 'test-results');
-        await promisify(fs.mkdir)(testResultsDirectory, { recursive: true });
-        await promisify(fs.writeFile)(
-            path.join(testResultsDirectory, 'index.html.axe-core.sarif'),
-            // We'll be checking in the resulting .sarif file for baselining purposes, so
-            // it's a good idea to use a spacing argument (here, "2") to pretty-print the
-            // JSON. This makes it much more pleasant to diff when it changes. 
-            JSON.stringify(sarifResults, null, 2));
-    });
-});
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { convertAxeToSarif } from 'axe-sarif-converter';
+import * as AxeBuilder from 'axe-webdriverjs';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Builder, By, ThenableWebDriver, until } from 'selenium-webdriver';
+import * as chrome from 'selenium-webdriver/chrome';
+import { promisify } from 'util';
+
+// The default timeout for tests/fixtures (5 seconds) is not always enough to start/quit/navigate a browser instance.
+const TEST_TIMEOUT_MS = 30000;
```

Comment:
nit: Do we want to move this out to a constant file and maintain constants there? Not a blocker.

Reply:
I would normally agree, but thought maybe keeping everything visible in context was more valuable in this particular case for sample purposes. Will consider moving this + some of the ChromeDriver setup into a separate setup file in a separate PR